tests/test_distributions.py

```diff
@@ -302,6 +305,18 @@ def test_expand_margins(self, flat_array):
         assert x1 == x2
         assert y1 + height * 2 == pytest.approx(y2)
 
+    def test_multiple_rugs(self, long_df):
+
+        values = np.linspace(start=0, stop=1, num=5)
+        ax = lineplot(x=values, y=values)
+        rugplot(x=values, ax=ax)
+        ylim = ax.get_ylim()
+
+        for j in range(4):
```

Comment:
Similar to above, calling rugplot once should trigger expansion even if it's a little hard to visually notice, so the loop is probably not necessary.

Reply:
Made the change 👍